2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,16 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(arr) {
+
+  let BestChoice= arr.filter(item =>
+    item.includes('%') &&
+    item.includes('.') &&
+    item > '19.5%' &&
```

Comment:
Do you think we should compare strings or numbers in the solution for this function?